src/gen/c_grammar.lua

```diff
@@ -156,7 +183,7 @@ local api_param_type = (
 local ctype = C(
   opt(word('const'))
     * (
-      typed_container
+      typed_container / 0
```

Comment:
(off topic) this is an example where [operator overloading](https://www.inf.puc-rio.br/~roberto/lpeg/#cap-num) is obfuscating more than helping

Reply:
I agree. The entirety of lpeg is like this. For common parsers like "optional" I've abstracted these away:

```lua
--- @param pat vim.lpeg.Pattern
local function opt(pat)
  return pat ^ -1
end

--- @param pat vim.lpeg.Pattern
local function rep(pat)
  return pat ^ 0
end

--- @param pat vim.lpeg.Pattern
local function rep1(pat)
  return pat ^ 1
end

local any = P(1)
```

Could do the same for `/ 0` but decided not to.